tests/unit_tests/udf_wrapper_params/token_classification/mock_sequence_tokenizer.py

```diff
@@ -2,5 +2,5 @@
 
 class MockSequenceTokenizer:
     @classmethod
-    def from_pretrained(cls, model_name, cache_dir, use_auth_token):
+    def from_pretrained(cls,cache_dir):
```

Comment:
this doesn't make sense, if this needs to change I would guess as following

```suggestion
    def from_pretrained(cls, model_name):
```

where model_name actully a path is. cache_dir is different parameter of from_pretrained which we don't really use anymore

Reply:
changed it to "model_path", this way it is consistent with the other unit tests.